### ISSUE TITLE
Two trees of packages in the spring-cloud-gcp-pubsub and spring-cloud-gcp-storage

### ISSUE THREAD

**Issue by artembilan** [spring-cloud/spring-cloud-gcp#373]
I think we need to move all the `integration` stuff into the swapped package.
For example right now we have these two trees in the `spring-cloud-gcp-pubsub`:
```
org.springframework.cloud.gcp.pubsub
...
org.springframework.cloud.gcp.integration.pubsub
```

That doesn't look good. Must be a single tree. Therefore like this: `org.springframework.cloud.gcp.pubsub.integration`.

The same for the `spring-cloud-gcp-storage`.

/CC @snicoll

**Comment by snicoll** [comment]
paging @dsyer 

What is the link between the two packages? Does the `pubsub`  package uses the `integration`  package or is it the the other  way around?

**Comment by dsyer** [comment]
The integration package uses pubsub I expect. I guess either we need to follow Artem's recommendation or add the "core" features in their own (sibling) package to "integration".

**Comment by joaoandremartins** [comment]
Fixed by #387
